src/databricks/labs/blueprint/cli.py

```diff
@@ -30,6 +31,15 @@ def needs_workspace_client(self):
             return False
         return True
 
+    def run_as_collection(self) -> bool:
```

Comment:
could you add a comment explaining how this would work, i.e. the command should have `is_collection` annotation & `collection_workspace_id` is not null

Reply:
also, our function would always receive `collection_workspace_id`, the difference is that it might be null instead of an integer right?

Reply:
This is my understanding. We mark a method as collection eligible by setting is_collection as True.
Now there are two ways to execute it , if the collection_workspace_id is passed, then create account context and iterate through all workspace context of the collection and run the associated command once for each workspace if collection_workspace_id is not passed, then run it as current installation cmd.

Reply:
added comments